cmux-tui/bindings/tests/test_verify_github_environment_policy.py

```
@@ -0,0 +1,162 @@
+            policy.EnvironmentPolicyError,
+            "api.github.com",
+        ):
+            policy._environment_url(
+                "https://collector.example.test",
+                "manaflow-ai/cmux",
+                self.environment_name,
+            )
+
+    def test_rejects_cross_origin_redirects(self) -> None:
+        request = policy.Request("https://api.github.com/repos/manaflow-ai/cmux")
+        with self.assertRaisesRegex(
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)